.github/workflows/pullrequest-lint.yml

```
@@ -33,3 +36,34 @@ jobs:
+  lint-changed-files:
+    name: Lint Changed Files
+    runs-on: arc-runners
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          # Need full repo for Nx to find the commits at NX_* below
+          fetch-depth: 0
+
+      - name: Derive appropriate SHAs
+        if: ${{ inputs.run_merge_queue == false }}
```

actionlint (1.7.4)

property "run_merge_queue" is not defined in object type {}

(expression)